### PR TITLE
Remove ProjectTemplates ownership

### PR DESCRIPTION
**Pull request by ryanbrandenburg**
Already done [in master](https://github.com/dotnet/aspnetcore/blob/master/.github/CODEOWNERS#L19). Being marked as CodeOwner for this area automatically puts my name as a reviewer on a lot of PRs on which I don't have useful input/context. People are always welcome to manually request my review if they believe my input would be useful.

### CODE REVIEW

File: .github/CODEOWNERS
Comment (dougbu) [inline .github/CODEOWNERS]: Remove, don't comment out

**Review by dougbu — CHANGES_REQUESTED** [review]
Once you've changed things, we can merge w/o waiting for the validation builds

File: .github/CODEOWNERS
Comment (ryanbrandenburg) [reply .github/CODEOWNERS]: The comment is done to keep things [in sync with master](https://github.com/dotnet/aspnetcore/blob/master/.github/CODEOWNERS#L19). I seem to remember that I originally was going to remove the entry but @mkArtakMSFT suggested I keep it as a comment to make sure it was still clear who "owned" the area.

@mkArtakMSFT am I remembering that correctly and/or what are your current opinions?

Comment (dougbu) [reply .github/CODEOWNERS]: Preference is **always** to remove lines rather than comment them out. If you want to add a clear comment like `# @ryanbrandenburg loves ProjectTemplates but not email notifications about it` that's fine but don't leave it looking like the "real" lines

**Review by dougbu — APPROVED** [review]

**Comment by dougbu** [comment]
nit: Suggest using a real comment in 'master' too. We're not doing automated merges from 'blazor-wasm' into 'master'

**Pull request merged**
